site/en/docs/privacy-sandbox/permissions-policy/index.md

```diff
@@ -0,0 +1,308 @@
+* If the response does not contain a Permissions Policy header, the origin list is considered to have the default value of `*`, and just adding the allow attribute to the iframe allows access to the feature.
+* Therefore, it is recommended that the Permissions Policy header is explicitly set in the response, so that cross-origin iframes that are not listed in the origin list will be blocked from accessing the feature even with an allow attribute in the tag.
+
+Feature Policy can still be used after Chrome 88, but it acts as an alias for Permissions Policy. Other than the syntax, there is no difference in logic between the two. If both Permissions Policy and Feature Policy are used together, the Permissions Policy header will have higher priority, and will overwrite the value provided from the Feature Policy header. 
+
+## Usage
+
+### Quick overview
+
+Before we dive deep, let’s take a quick look at a common scenario where you are the owner of a website and you want to control how your site and third-party code use browser features. 
+
+* Your site is `https://your-site.example`.
+* Your site embeds an iframe from same-origin (`https://your-site.example`). 
+* Your site embeds an iframe from `https://trusted-site.example` that you trust. 
+* Your site also displays ads served by `https://ad.example`. 
+* You want to enable the geolocation feature only for your site and the trusted site. 
+
+In this case, use the following header: 
+```text
+Permissions-Policy: geolocation=(self "https://trusted-site.example")
+```
+
+And explicitly set the `allow` attribute in the iframe tag: 
+```html
+<iframe src="https://trusted-site.example" allow="geolocation">
+```
+{% Img src="image/hVf1flv5Jdag8OQKYqOcJgWUvtz1/6glnEtP01JOyxOtdephq.png", alt="Quick overview diagram of Permissions Policy usage", width="800", height="238" %}
+
+In this example, the header origin list lets only your site (`self`) and `https://trusted-site.example` to use the geolocation feature. `https://ad.example` is not allowed to use geolocation. 
+
+1. Your code from `https://your-site.example` is allowed to use the geolocation feature with the user’s consent.
+1. An iframe served from same-origin (`https://your-site.example`) is allowed to use the feature without the usage of the `allow` attribute.
+1. An iframe served from a cross-origin (`https://trusted-site.example`) that has been added to the origin list, and also has the `allow` attribute set on the iframe tag is allowed to use the feature.
+1. An iframe served from a cross-origin (`https://trusted-site.example`) that has been added to the origin list, but does not have the `allow` attribute set on the iframe tag is blocked from using the feature.
+1. An iframe served from a cross-origin (`https://ad.example`) that has not been added to the origin list is blocked from using the feature even with the `allow` attribute on the iframe tag.
+
+### HTTP Header
```

Comment:
Hmm, this section is more about how the Permissions Policy header works, rather than telling them to add it.  I'm wondering if "`Permissions-Policy` HTTP response header" is better.  Let me know.